db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt

```diff
@@ -118,6 +118,16 @@ data class ConnectionPoolConfiguration @JvmOverloads constructor(
         coroutineDispatcher = coroutineDispatcher
     )
 
+    override fun toString() = "ConnectionPoolConfiguration(host=$host, port=REDACTED, database=$database, " +
+            "username=REDACTED, password=REDACTED, maxActiveConnections=$maxActiveConnections, " +
+            "maxIdleTime=$maxIdleTime, maxPendingQueries=$maxPendingQueries, " +
+            "connectionValidationInterval=$connectionValidationInterval, " +
+            "connectionCreateTimeout=$connectionCreateTimeout, connectionTestTimeout=$connectionTestTimeout,\n" +
+            "queryTimeout=$queryTimeout, eventLoopGroup=$eventLoopGroup, " +
```

Comment:
Why do we need to stringify eventLoopGroup

Reply:
fixed.